tools/doc/hawqregister_help

```diff
@@ -0,0 +1,101 @@
+COMMAND NAME: hawq register
+
+Register parquet files generated by other system into the corrsponding table in HAWQ
+
+*****************************************************
+SYNOPSIS
+*****************************************************
+
+hawq register [-h hostname] [-p port] [-U username] <databasename> <tablename> <hdfspath>
+
+hawq register help
+hawq register -?
+
+hawq register --version
+
+*****************************************************
+DESCRIPTION
+*****************************************************
+
+"hawq register" is a utility to register file(s) on HDFS into
+the table in HAWQ. It moves the file in the path(if path
+refers to a file) or files under the path(if path refers to a
+directory) into the table directory corresponding to the table,
+and then update the table meta data to include the files.
+
+To use "hawq register", HAWQ must have been started.
+
+Currently "hawq register" supports parquet tables only.
+User have to make sure that the meta data of the parquet file(s)
+and the table are consistent.
+The table to be registered into should not be hash distributed, which
+is created by using "distributed by" statement when creating that table.
+
+*****************************************************
+Arguments
+*****************************************************
+
+<databasename>
+
+Name of the database to be operated on.
+
+<tablename>
+
+Name of the table to be registered into.
+
+<hdfspath>
+
+The path of the file or the directory containing the files
+that will be registered.
+
+*****************************************************
+OPTIONS
+*****************************************************
+
+-? (help)
+
+Displays the online help.
+
+--version
+
+Displays the version of this utility.
+
+*****************************************************
+CONNECTION OPTIONS
+*****************************************************
+
+-h hostname
+  
+  Specifies the host name of the machine on which the HAWQ master
+  database server is running. If not specified, reads from the
+  environment variable $PGHOST or defaults to localhost.
```

Comment:
What's the meaning for "or"? It may confuse user.  I think we can change "or" to "which".